tests/ellipsis_test.py

```diff
@@ -373,23 +374,26 @@ def testKwargsMustBeUsedAsStarKwarg(self):
             when(rex).bark(*kwargs).thenReturn('Miau')
 
     def testNiceFormattingForEllipsis(self):
-        m = mock()
+        obj = mock()
+        m = mock_registry.mock_for(obj)
```

Comment:
These were actually correct.  We use a fake object (aka `mock({"strict": False)`) as the first argument to `StubbedInvocation`.